### PR TITLE
Support PHP-Parser v4

### PR DESCRIPTION
**Pull request by carusogabriel**
Looks like no code change was needed :smiley:

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
         "php": "7.*",
         "ext-mbstring": "*",
         "ext-tokenizer": "*",
-        "nikic/php-parser": "^2.1|^3.0"
+        "nikic/php-parser": "^4.0"
```
Comment (jdreesen) [inline composer.json]: Why don't you just add `^4.0` to `^2.1|^3.0` instead of replacing it, if it's compatible with all three major versions?

Comment (carusogabriel) [reply composer.json]: v2 makes no sense to have it as we minimum require PHP 7, but we can stay with v3 :smile:

**Comment by carusogabriel** [comment]
Should be RTM :)

**Comment by assertchris** [comment]
Included in https://github.com/marcioAlmada/yay/pull/51

**Comment by carusogabriel** [comment]
Merging this one.

@assertchris Thanks, but feel free to rebase on top of master now :smile:

**Pull request merged**